applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/MaterialParameters1.json

```diff
@@ -22,8 +22,7 @@
           "YOUNG_MODULUS": 100000.0,
           "POISSON_RATIO": 0.2,
           "K0_MAIN_DIRECTION": 1,
-	      "NUMBER_OF_UMAT_PARAMETERS":  6,
-          "INDEX_OF_UMAT_PHI_PARAMETER": 4,
+	      "INDEX_OF_UMAT_PHI_PARAMETER": 4,
```

Comment:
Nitpicking: perhaps we can make the level of indentation consistent with the previous and next lines?